### PR TITLE
Update README.rst to link to Python Getting Started sample and tutorial.

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -58,8 +58,11 @@ For more information, see `Supported Python Versions`_ in
 Example Applications
 --------------------
 
+-  `getting-started-python`_ - A sample and `tutorial`_ that demonstrates how to build a complete web application using Cloud Datastore, Cloud Storage, and Cloud Pub/Sub and deploy it to Google App Engine or Google Compute Engine.
 -  `gcloud-python-expenses-demo`_ - A sample expenses demo using Cloud Datastore and Cloud Storage
 
+.. _getting-started-python: https://github.com/GoogleCloudPlatform/getting-started-python
+.. _associated tutorial: https://cloud.google.com/python
```
Comment (jgeewax) [inline README.rst]: Is this text right? `s/_associated tutorial:/_tutorial:/` ?

Comment (dhermes) [inline README.rst]: :+1:

Comment (jonparrott) [inline README.rst]: fixed.

**Comment by dhermes** [comment]
LGTM (LMK if you don't have merge privileges)

**Comment by jonparrott** [comment]
Thanks. I do not have merge powers.

**Pull request merged**